proteus/mprans/CLSVOF.h

```diff
@@ -878,8 +881,9 @@ namespace proteus
 		    // SAVE SOLUTION // for other models
 		    ///////////////////
 		    q_u[eN_k] = u;
-		    q_m[eN_k] = u; //porosity*u;
+		    q_m[eN_k] = u;
```

Comment:
Shouldn't this be `q_m[eN_k] = porosity*u'`?

Reply:
I would think so @manuel-quezada. Based on the history, you removed the comment that suggests that, however. I think the main thing is that we be clear and consistent about which q fields store the vof and the sdf.

Reply:
Looking at the whole file and how this variable is used, I actually think it should be porosity*Hnp1. This should be the conserved quantity, which is actually porosity*H(u).  I don't think we're using this variable in the actual CLSVOF method because of how the time discretization is actually implemented.

Reply:
I switched this to porosity*u and tried to make the additional changes required by that.